### DISCUSSION TITLE
제페토 월드 맵 제작시 문의사항입니다

### DISCUSSION THREAD

**Discussion by sunysiny** [naverz/zepeto-studio-kor#695] [5. Q&A]
안녕하세요:)
유니티에서 월드맵을 제작 중에 몇가지 문의사항이 있어서 질문 남깁니다.

1.허용되는 패스콜의 최대 수치와 일반적인 맵의 패스콜 수치는 어느정도인가요?

2. 사용하여야 하는 머테리얼과 쉐이더의 종류는 아바타 제작시와 같은 것을 사용하면 되나요?

3. 지원하는 텍스쳐의 최대 크기(사이즈)와 용량 크기는 어떻게 되는지?
  (최대크기가 어느정도냐에 따라 아틀라스를 함에 따라 1024까지 사용할 수 있는지에 대함도)

4. 자체 제작 이미지 사용 스카이박스를 사용할 시 최대 사이즈와 용량 (압축-32비트레이트 등의 세부적인 옵션 조절을 하여 사용이 가능한가요?)

5. 월드 하나에 사용이 가능한 최대 트리스 제한 수 (아바타처럼 기준 수치가 있나요?)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 말씀하신 그래픽 관련 제한은 없습니다.
저사양 단말에서 동작이 되는 것이 중요합니다. 아이폰 6, 갤럭시 S6 기기를 보통 저사양 단말로 보고 있습니다.
감사합니다.
